wormhole-connect/src/sdklegacy/types.ts

```diff
@@ -22,17 +22,13 @@ export type ChainConfig = {
   key: Chain;
   id: ChainId;
   context: Context;
-  finalityThreshold: number;
   disabledAsSource?: boolean;
   disabledAsDestination?: boolean;
 };
 
 export type WormholeConfig = {
   env: Network;
   rpcs: ChainResourceMap;
-  rest: ChainResourceMap;
```

Comment:
Removed these legacy configs which are unused: `rest`, `graphql`, and `wormholeHosts`.